src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,10 @@
+@mixin square($size) {
+  width: $size;
+  height: $size;
+  border-radius: $border-radius;
```

Comment:
The $border-radius variable is not defined or passed as an argument in the mixin. You should pass it as an argument to make the mixin more reusable.